sample_app/app/views/users/_form_user.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= form_for user do |f| %>
-  <%= render "shared/error_messages", object: user %>
+  <%= render "shared/error_messages", object: f.object %>
```

Comment:
cách này truyền object vào error_messages, truyền object có thể dùng linh hoạt cho cả micropost form error
- với form for user thì object là user ( TH này)
- với form for micropost thì object là micropost ạ